userguide/storage.rst

```diff
@@ -1198,8 +1198,8 @@ available when editing multiple disks.
    |                              | menu      |            | `AAM <https://en.wikipedia.org/wiki/Automatic_acoustic_management>`__.                                                   |
    |                              |           |            |                                                                                                                          |
    +------------------------------+-----------+------------+--------------------------------------------------------------------------------------------------------------------------+
-   | Enable S.M.A.R.T.            | checkbox  | ✓          | Enabled by default if the disk supports S.M.A.R.T. Unsetting this option will disable any configured                     |
-   |                              |           |            | :ref:`S.M.A.R.T. Tests` for the disk.                                                                                    |
+   | Enable S.M.A.R.T.            | checkbox  | ✓          | Enabled by default when the disk supports S.M.A.R.T. Unsetting disables any configured disk :ref:`S.M.A.R.T. Tests` and  |
+   |                              |           |            | prevents gathering disk temperature data for system :ref:`reports <Reporting>`.                                          |
```

Comment:
"Disabling S.M.A.R.T. tests prevents collecting new temperature data for this disk. Historical temperature data is still displayed in :ref:`Reporting`."